test/e2e/common/security_context.go

```diff
@@ -227,27 +227,27 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		}
 
 		/*
-		  Testname: allowPrivilegeEscalation unset and uid != 0.
-		  Description: Configuring the allowPrivilegeEscalation unset, allows the privilege escalation operation.
-		  A container is configured with allowPrivilegeEscalation not specified (nil) and a given uid which is not 0.
-		  When the container is run, the container is run using uid=0.
-		  This test is marked LinuxOnly since Windows does not support running as UID / GID, or privilege escalation.
+			Release : v1.14
+			Testname: Security Context, allowPrivilegeEscalation unset, uid != 0.
+			Description: Configuring the allowPrivilegeEscalation unset, allows the privilege escalation operation.
+			A container is configured with allowPrivilegeEscalation not specified (nil) and a given uid which is not 0.
+			When the container is run, container's output MUST match with expected output verifying container ran with uid=0.
 		*/
-		It("should allow privilege escalation when not explicitly set and uid != 0 [LinuxOnly] [NodeConformance]", func() {
+		framework.ConformanceIt("should allow privilege escalation when not explicitly set and uid != 0 [NodeConformance]", func() {
```

Comment:
I don't agree that this should be a conformance test.  This is dependent on the runtime behavior of the container engine.  The image itself would have to be part of the conformance test, and not all conformant platforms will require this behavior.

Reply:
Please do not remove the ``[LinuxOnly]`` label, test contains a SecurityContext, which is not working on Windows.

Reply:
Not Promoting this e2e to Conformance. Noted reason in the code comment.